.vscode/settings.json

```
@@ -0,0 +1,5 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "editor.formatOnPaste": false
+}
```

ref .eslintrc.json:L48 `"eol-last": ["error", "always"],`
```suggestion
}

```